src/definitions/modules/embed.js

```diff
@@ -612,7 +612,7 @@
 
         templates: {
             escape: function (string) {
-                const badChars = /["'<>]|&(?![\d#A-Za-z]{1,12};)/g;
+                const badChars = /["&'<>]/g;
```

Comment:
What about making this regEx a setting parameter like "regEx.escape" like we have in search, form and dropdown? (default /["&'<>]/g)
This way, the previous approach (allowing to not encode existing htmlentities), would be at least possible to adjust by the user via

```javascript
.dropdown({
regExp: {
    escape: /["'<>]|&(?![\d#A-Za-z]{1,12};)/g
}
})
```

Reply:
I am strongly againts. The reason is such function/regex should never be changed by the developer. And as discussed in https://github.com/fomantic/Fomantic-UI/pull/3205#discussion_r2014886407 and each commit message, the original regexes are only helpful for input data from innerHTML. This is the case only in 1 single usage in dropdown.

If user want to call FUI API with data from HTML, he must deescape the input like https://github.com/fomantic/Fomantic-UI/pull/3231/commits/17dd1fd017a521994a570b213eed0302e3000b40.